kyuubi-common/src/main/scala/org/apache/kyuubi/engine/schema/AbstractTRowSetGenerator.scala

```diff
@@ -16,28 +16,20 @@
  */
 
 package org.apache.kyuubi.engine.schema
```

Comment:
the `schema` seems incorrect to me
```suggestion
package org.apache.kyuubi.engine.result
```

Reply:
Changed.